samples/plugins/mysql_odbc/manifest.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version='1.0' encoding='utf-8' ?>
+
+<connector-plugin class='mysql_odbc_connector' superclass='mysql_odbc' plugin-version='0.0.0' name='MySQL ODBC Plugin Connector' version='18.1'>
```

Comment:
I think we should start aligning around adding 'sample' to our samples, since they get copy/pasted to start a lot.

class='mysql_odbc_sample' ... name='Sample MySQL ODBC'

Reply:
Good idea, I'll fix that here and open a separate PR to add `sample` to the other samples.